### PR TITLE
Added keypoint matplotlib viewer

### PR DESCRIPTION
**Pull request by hausmanns**
Matplotliib viewer to view your keypoints' trajectory synced with the video and keypoints.

Activate plugin, upload your .h5 (and video, not necessary), and tick "show matplotlib trajectory"

- [x] Needs a fix: For now it's still hardcoded to the naming of my keypoints of interests.
Needs to be adapted to keypoints from the GUI directly, in here:

image

https://github.com/DeepLabCut/napari-deeplabcut/assets/37653619/2cc70a56-f8ef-4c0c-a252-d133054e6193

### CODE REVIEW

**Comment by jeylau** [comment]
https://github.com/DeepLabCut/napari-deeplabcut/assets/30733203/ae3699c3-f08a-41f2-9c8f-d4fb6623fee1

**Comment by jeylau** [comment]
Probably a couple of tweaks that could be made here and there; but that works very smoothly even for maDLC.

**Comment by hausmanns** [comment]
Another thing that should be tweaked @jeylau :

They y_lim should be independent of the dataset and dependent on the plot window (or adjustable)
Indeed, keypoint jumps can squeeze the window depending on the dataset's quality

**Comment by jeylau** [comment]
Good point; one thing that could be added then is a `NavigationToolbar2QT`, which would allow panning, zooming, etc.

**Comment by MMathisLab** [comment]
Is this superseded by #92 ?

**Comment by jeylau** [comment]
@MMathisLab, these 2 PRs are distinct components. I will add the toolbar Seb mentioned above so this one is ready 😊

**Comment by jeylau** [comment]
Also, regarding the warnings in the console, they will be removed with napari's next release and vispy >= 0.13 https://github.com/napari/napari/pull/6025

**Comment by MMathisLab** [comment]
I think we need to address failing tests here before merge, right?

**Comment by jeylau** [comment]
Mhhh, napari removed the plugin preview service https://github.com/chanzuckerberg/napari-hub/discussions/1216. I'll delete the github action.

**Comment by MMathisLab** [comment]
so, ready to merge?

**Comment by jeylau** [comment]
Yes, looking good to me!

**Comment by MMathisLab** [comment]
can you review it then, so we can merge?

File: setup.cfg [hunk 1]
```diff
@@ -33,6 +33,7 @@ project_urls =
 packages = find:
 install_requires =
     dask-image
+    matplotlib
```
Comment (MMathisLab) [inline setup.cfg]: isn't this installed with DLC proper, and we should be mindful of version?

**Review by MMathisLab — CHANGES_REQUESTED** [review]

**Review by MMathisLab — APPROVED** [review]

**Pull request merged**